authentication/config/locales/en.yml

```diff
@@ -23,8 +23,8 @@ en:
     sessions:
       new:
         hello_please_sign_in: Hello! Please sign in.
-        sign_in: Sign in
         forgot_password: I forgot my password
+        sign_in: Sign in
```

Comment:
I was sorting the italian locale mirroring the english one... I must have confused them here. One moment.